### PR TITLE
Fix Add enabled currencies modal UI bug

### LINKED ISSUE #2731: Add enabled currencies modal UI bug
The modals we're using in multi-currency and UPE are experiencing a UI bug when the Gutenberg extension is active: 

with gutenberg plugin active 

Screen Shot 2021-08-13 at 10 28 13 AM

Without gutenberg plugin active

Screen Shot 2021-08-13 at 10 30 54 AM

Even in the case where gutenberg is not active, we're still getting clipping on the focus state of the selected item

cc @frosso since you looked at this issue previously

### PR DESCRIPTION
**Pull request by oaratovskyi**
Fixes #2731 

#### Changes proposed in this Pull Request

Fixed bugs with UI modal:
- Fixed styling of the header _Add enabled currencies_ by overriding `left: auto`. In Gutenberg plugin there is a `left: 0` by default.
- Fixed margin of the checkbox (`.components-checkbox-control__input`) from `margin: 0 4px 0 0` to `margin: 0 4px 0 4px`. It allows the `box-shadow` to become visible.
- Fixed svg `left` value from `-2px` to `2px` on desktop and `4px` on mobile due to margin fix in the previous stage.

#### Screenshots

After fix (with activated Gutenberg plugin) - Desktop
image

Mobile
image

#### Testing instructions

* Enable the Gutenberg plugin
* Check that enabled currencies modal looks according to the screenshot (or the connected issue)
* Disable the Gutenberg plugin
* Check that enabled currencies modal still looks according to the screenshot (or the connected issue)

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)

### CODE REVIEW

File: client/multi-currency/enabled-currencies-list/style.scss [hunk 1]
```diff
@@ -183,5 +189,10 @@
 			min-height: 0;
 			flex: 1;
 		}
+		.enabled-currency-checkbox {
+			.components-checkbox-control__checked {
+				left: 4px;
+			}
+		}
```
Comment (ismaeldcom) [inline client/multi-currency/enabled-currencies-list/style.scss:196]: It works, but maybe we can simplify this by adding just `padding-left: 4px` on `.add-enabled-currencies-modal &__content`.

**Review by ismaeldcom — APPROVED** [review]
LGTM! Works as expected, and solves the Gutenberg issue. Left a tiny comment about another approach to solve the checkbox shadow cut. But I'm preapproving it 🙂 

Let's 🚢  it!

File: changelog.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@
 = 3.0.0 - 2021-xx-xx =
 * Add - Download deposits report in CSV.
 * Fix - Use store currency on analytics leaderboard when Multi-Currency is enabled.
+* Fix - Add enabled currencies modal UI.
```
Comment (ismaeldcom) [inline changelog.txt:6]: Nitpick here 🙂 I think might be better to describe the actual fix, cause looks like you're adding the modal for the first time. Maybe just removing the `Add ` is enough.

**Review by LevinMedia — APPROVED** [review]

File: changelog.txt (hunk 2 above)
Comment (oaratovskyi) [reply changelog.txt]: Makes sense 🙂
Done in https://github.com/Automattic/woocommerce-payments/pull/2840/commits/aa25a0d07d680278e9a36645e6ca3b894bd53036

File: client/multi-currency/enabled-currencies-list/style.scss (hunk 1 above)
Comment (oaratovskyi) [reply client/multi-currency/enabled-currencies-list/style.scss]: Great suggestion, that's simplifies the code a lot.
Thanks!
Fixed in https://github.com/Automattic/woocommerce-payments/pull/2840/commits/d0f1903dfcd556e40599578b84511d3eea424f8d

**Pull request merged**